### ISSUE TITLE
ASL Registration 

### ISSUE THREAD

**Issue by tsawyer** [AllStarLink/app_rpt#48]
The ASL registration system consists of a home-grown nodejs registration server and a modified chan_iax. We did this because Realtime Asterisk couldn't keep up with our 1,000's of registrations. Couldn't get reliably above 1,000 or so. We're over 8,000 at this point in time.

The ASL 2.0 beta modified chan_iax uses both http and IAX. It will try http first, as that is preferred, and fall back to IAX.   

Bottom line is a modified chan_iax will never make it into the Asterisk source. A new registration method will need to be developed for the client.  The nodejs registration server is open source and could be placed into the Asterisk source tree.
